elasticai/creator/nn/vhdl/identity/design.py

```diff
@@ -27,7 +25,7 @@ def port(self) -> Port:
         )
 
     def save_to(self, destination: Path) -> None:
-        template = Template(
+        template = InProjectTemplate(
```

Comment:
Probably better left up for another pull request, but:

Since we have software components mirroring hardware designs i want to keep both in one place. I fear that might render the `InProjectTemplate` useless. See also PR #238